libs/langchain/tests/unit_tests/stubs.py

```diff
@@ -20,28 +20,28 @@ def __eq__(self, other: object) -> bool:
 # subclassed strings.
 
 
-def _AnyIdDocument(**kwargs: Any) -> Document:
+def any_id_document(**kwargs: Any) -> Document:
```

Comment:
I'd prefer to keep these as they were as they stand in for replacement classes

Reply:
I opened https://github.com/langchain-ai/langchain/pull/32209 .
LMK what you think.

Reply:
done.